tests/scadnano_tests.py

```diff
@@ -604,8 +604,8 @@ def test_32_helix_rectangle(self) -> None:
         # To help with debugging, uncomment these lines to write out the
         # scadnano file
         #
-        # design.write_scadnano_file(directory=self.output_path,
-        #                            filename=f'test_32_helix_rectangle.{sc.default_scadnano_file_extension}')
+        design.write_scadnano_file(directory=self.output_path,

```

Comment:
would it be possible to leave these `write_scadnano_file` lines commented? They are only there for debugging purposes. We don't want unit tests to write files that we don't need.